Recrutify/Recrutify.Host/Configuration/Profiles/ProjectProfile.cs

```diff
@@ -13,7 +13,7 @@ public ProjectProfile()
                 .ForMember(dest => dest.Id, conf => conf.MapFrom(src => Guid.NewGuid()));
             CreateMap<Project, ProjectDTO>().ReverseMap();
             CreateMap<StaffDTO, Staff>().ReverseMap();
-            CreateMap<PrimarySkill, PrimarySkillDTO>().ReverseMap();
```

Comment:
Return this map CreateMap<PrimarySkill, PrimarySkillDTO>(). It's used in primary-skills endpoint